src/integrationtest/python/doc_traversals_tests.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from janusgraph_container import JanusGraphContainer
+from janusgraph_python.driver.janusgraph_remote_connection_builder import JanusGraphRemoteConnectionBuilder
+from gremlin_python.structure.graph import Graph
+
+
+class TestDocTraversals(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        self.client = JanusGraphRemoteConnectionBuilder().connect(host=self.container.get_host_ip(), port=8182,
+                                traversal_source="gods_traversal").get_connection()
+
+        self.container.start()
+        self.g = Graph().traversal().withRemote(self.client)
+
+    def tearDown(self) -> None:
+        self.client.close()
+        self.container.stop()
+
+    def test_gremlin_getting_started(self):
+
```

Comment:
Remove extra blank lines within these test methods, please.

Reply:
Do you mean we should start `code` right after function definition? I am removing blank lines before `return` statement, but starting the code right after function starts, without a blank won't that reduce the **legibility** of code?
As for blank line between query and assert statement, I guess that's okay as those 2 are essentially different logic blocks right?